.github/workflows/export.yml

```diff
@@ -8,12 +8,34 @@ on:
         description: 'Also build a Flatpak bundle'
         type: boolean
   pull_request:
+    paths-ignore: &paths_ignore
+      - '**.aseprite'
+      - '**.license'
+      - 'LICENSES/**'
+      - 'docs/**'
+      - 'linux/**'
```

Comment:
Note to self: remove this in light of the change in #1299 to automatically build the flatpak on changes in this directory!